### PR TITLE
H2 upgrade docs

### PR DESCRIPTION
**Pull request by zhmaeff**

### CODE REVIEW

File: docs/modules/deploy-manage/pages/h2-upgrade.adoc [hunk 1]
```diff
@@ -0,0 +1,19 @@
+= H2 upgrade
+:description: Management Center uses H2 database to store some metadata like connected cluster, security provider config, client filtering configs, etc. Starting from 5.4.0 Management Center uses new version of H2 database. Unfortunately metadata from the previous H2 version we used is not binary compatible with the new H2 version. That is why data migration is required.
```
Comment (promanenko) [inline docs/modules/deploy-manage/pages/h2-upgrade.adoc:2]: connected cluster -> connected clusters

Comment (promanenko) [inline docs/modules/deploy-manage/pages/h2-upgrade.adoc:2]: I think it makes sense to explain why we HAD to migrate despite compatibility issues

File: docs/modules/deploy-manage/pages/h2-upgrade.adoc [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+== Seamless H2 data migration
+
+Starting with 5.4.0 we deliver additional `h2-upgrade-cli-*.jar` that is impicitly invoked during Management Center startup in `./bin/hz-mc start` command. So if you use bundled `hz-mc` tool or `hazelcast/management-center` docker image to start Management Center, no actions required on your side, H2 data will be automatically backed up and migrated.
+
+== H2 data migration
+
+If in your setup you have to use custom scripts to run Management Center and you don't call `./bin/hz-mc start` from it, you need to upgrade and migrate H2 data manually.
+
+There is a new command `hz-mc upgrade-h2` that runs all migrations for you.
```
Comment (promanenko) [inline docs/modules/deploy-manage/pages/h2-upgrade.adoc:14]: please add info about the backup file we create for debugging purposes and that customers can safely delete it when migration succeeded

**Review by promanenko — COMMENTED** [review]
Overall LGTM

File: docs/modules/deploy-manage/pages/h2-upgrade.adoc (hunk 2 above)
Comment (zhmaeff) [reply docs/modules/deploy-manage/pages/h2-upgrade.adoc]: Updated, please check

**Review by promanenko — APPROVED** [review]

**Review by rebekah-lawrence — APPROVED** [review]
LGTM

**Pull request merged**